docker-compose.yml

```diff
@@ -31,6 +30,17 @@ services:
       database-init:
         condition: service_completed_successfully
 
+  frontend:
+    build:
+      context: ./frontend
+    command: npm run start
+    ports:
+      - "3210:3000"
+    working_dir: /app
+    volumes:
+      - ./frontend:/app # bind-mount source code
```

Comment:
Don't bind-mount the source code in the production Compose file. Do that in the `docker-compose.dev.yml` file instead.

```suggestion
```